tests/JustSaying.TestingFramework/Patiently.cs

```diff
@@ -6,10 +6,20 @@ namespace JustSaying.TestingFramework
 {
     public static class Patiently
     {
-        public static async Task AssertThatAsync(Func<bool> func)
-            => await AssertThatAsync(func, 5.Seconds()).ConfigureAwait(false);
+        public static async Task AssertThatAsync(
+            Func<bool> func,
+            [System.Runtime.CompilerServices.CallerMemberName]
+            string memberName = "")
+            => await AssertThatAsyncInner(func, 5.Seconds(), memberName).ConfigureAwait(false);
 
-        public static async Task AssertThatAsync(Func<bool> func, TimeSpan timeout)
+        public static async Task AssertThatAsync(
+            Func<bool> func,
+            TimeSpan timeout,
+            [System.Runtime.CompilerServices.CallerMemberName]
+            string memberName = "")
```

Comment:
Nice one, was scratching my head over the call site

Reply:
Annoyingly it doesn't seem to have helped - in fact we no longer log to the console at all for some reason.

https://travis-ci.org/github/justeat/JustSaying/jobs/713388237

Reply:
Seems like it's getting stuck/deadlocking somewhere.

Reply:
It's a head-scratcher. I've been playing about in my own branch and finding if I explicitly increase the threadpool min thread count it locks up immediately (on the unit test project, before it even gets to integration tests).
https://travis-ci.org/github/slang25/JustSaying/jobs/713977574
which I can't replicate locally.
I'm not really sure why 🤔